src/account/SupportContact.tsx

```diff
@@ -92,23 +67,27 @@ function SupportContact({ route }: Props) {
       body: `${message}<br/><br/><b>${JSON.stringify(deviceInfo)}</b>`,
       isHTML: true,
     }
-    let logsPath: string | false = false
+    let attachments: Array<Email['attachments']> | any
```

Comment:
i have some ideas about how to resolve the TS errors without `any`
1. this type should be `Email['attachments']` right? because `Email['attachments']` is already an array?
2. the return type of `Logger.getLogsToAttach()` should be `Email['attachments']` i think, rather than ``Email['attachments']` | undefined` - to do this i think you can bring the initialisation of `toAttach` in `getLogsToAttach` to the top line, and always return this value (i.e. return outside of the try/catch). so if there is an error, then an empty array is returned (which i think is not wrong)
3. if this is all fine, the we don't need to check for `if(attachments)` anymore, since it will always be an array
4. then in the `sendEmail` argument, we can have `attachments.find(
          (attachment: { name: string }) => attachment.name === Logger.getCurrentLogFileName()
        )?.path ?? false`

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2680/commits/ded87a8f61a5eeb9e9da0df52cc1691b823478c7!